libs/blocks/merch/merch.js

```diff
@@ -40,8 +40,7 @@ export function getTacocatEnv(envName, locale) {
 
   const literalScriptUrl = `${host}/special/tacocat/literals/${language}.js`;
   const scriptUrl = `${host}/special/tacocat/lib/${VERSION}/tacocat.js`;
-  const tacocatEnv = envName === ENV_PROD ? 'PRODUCTION' : 'STAGE';
-  return { country, language, literalScriptUrl, scriptUrl, tacocatEnv };
+  return { country, language, literalScriptUrl, scriptUrl, tacocatEnv: 'PRODUCTION' };
```

Comment:
Since this is always 'PRODUCTION', does it even make sense to define it here? Could it not be just a constant at the top of the file, which you can then just reference when needed? This would avoid passing a static value as an argument through a chain of functions